tests/stokes_01.output

```diff
@@ -1,43 +1,43 @@
 
-DEAL:FGMRES::Starting value 26.4144
-DEAL:FGMRES::Convergence step 7 value 2.86961e-15
-DEAL:FGMRES::Starting value 2.81099e-14
-DEAL:FGMRES::Convergence step 0 value 2.81099e-14
-DEAL:FGMRES::Starting value 2.81099e-14
-DEAL:FGMRES::Convergence step 0 value 2.81099e-14
+DEAL:FGMRES::Starting value 4.21364
+DEAL:FGMRES::Convergence step 3 value 7.04932e-12
+DEAL:FGMRES::Starting value 7.04918e-12
+DEAL:FGMRES::Convergence step 0 value 7.04918e-12
+DEAL:FGMRES::Starting value 7.04918e-12
+DEAL:FGMRES::Convergence step 0 value 7.04918e-12
```

Comment:
si possono non far stampare queste iterazioni? mettendo 0 al posto di 2 in stokes_01.cc?
magari come output di confronto possiamo mettere la norma della soluzione. ho timore che questi valori facciano fallire i test su computer diversi. @luca-heltai ?